CONTRIBUTING.md

```diff
@@ -46,19 +46,24 @@ $ ./phpunit
 
 ### JavaScript Test Suite
 
-First, install [PhantomJS](http://phantomjs.org/) and [Google Closure
-Library](https://github.com/google/closure-library):
+First, install [PhantomJS](http://phantomjs.org/) (see the website for further 
+details or simply use your favourite package manager) and the [Google Closure
+Library](https://github.com/google/closure-library) using:
 
 ```bash
+$ cd Resources
 $ npm install google-closure-library
 ```
 
 Run the JS test suite with:
 
 ```bash
-$ phantomjs Resources/js/run_jsunit.js Resources/js/router_test.html
+$ phantomjs js/run_jsunit.js js/router_test.html
```

Comment:
I suggest adding this command as the `test` script in package.json, so that the standard `npm test` command can be used.

Reply:
Good call, done in a soon-to-be pushed commit. Documentation also updated.